core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -261,7 +261,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                                             eventLoop, id, 1, headers, false, inboundTrafficController,
                                             serviceConfig.maxRequestLength(), routingCtx,
                                             ExchangeType.BIDI_STREAMING,
-                                            System.nanoTime(), SystemInfo.currentTimeMicros(), true, false);
+                                            System.nanoTime(), SystemInfo.currentTimeMicros(), true, false
+                                    );
```

Comment:
Perhaps we can revert this cosmetic change?
```suggestion
                                            System.nanoTime(), SystemInfo.currentTimeMicros(), true, false);
```

Reply:
Thanks addressed. 😉